dao/build.gradle

```diff
@@ -0,0 +1,18 @@
+dependencies {
+    compile project(':domain')
+    compile group: 'org.springframework', name: 'spring-jdbc', version: '5.3.3'
+    compile group: 'org.springframework', name: 'spring-webmvc', version: '5.3.3'

```

Comment:
I believe WEB dependencies should not be used in DAO layer.

Reply:
my bad. sorry for that. removed